shared/src/test/diff/codegen/Polyfill.mls

```diff
@@ -50,10 +52,50 @@ n = 123 with { x = 1 }
 //│ n: 123 & {x: 1}
 //│  = [Number: 123] { x: 1 }
 
-:js
-n + 1
```

Comment:
Why remove the `n + 1` test? I think it's also useful.

Reply:
It was back!